### DISCUSSION TITLE
Git Desktop/VSCode dont synchronize with GitHub profile.

### DISCUSSION THREAD

**Discussion by mystertaran** [community/community#37871] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone.

For several days I have the following problem with my repositories - all changes in the repositories approved in VSC or by GIT Desktop are approved (I can see all changes in History for example), but they do not show up on the GitHub profile. Also all changes are not saved on my profile - for do that I have to upload all files where I made some changes.

Of course before I wrote this issue I alredy compare email adresses, I checked master branch and I dont saw reason why its not working as it should.

Could someone know how to resolve this issiue?

**Comment by jfullstackdev** [comment]
can you have the screenshot of these things? so we can take a look also

**Reply by mystertaran** [reply]
Yes, sure.

There you can see screenshot from history inside Git Desktop. Even if I made some commits via VSC its also made inside Git Desktop so they are synchronized as it should. There is some synchro problem betwen my desktop and Git Hub profile...
![Bez tytułu](https://user-images.githubusercontent.com/113885422/199075209-75fd5164-6e13-4bc6-8be9-a16a53e14168.png)

Please, compare it with screen below:

![2](https://user-images.githubusercontent.com/113885422/199075740-534de731-2b73-49fb-aa83-9eb65dde9927.png)
This contribution I have to made by using "Drag & drop" directly on my repository.

**Comment by jfullstackdev** [comment, marked as answer]
as you can see there, it's not published on GitHub, it's just a local repo 

![image](https://user-images.githubusercontent.com/47092464/199150419-2edd4bc6-8c86-49e8-a2d6-fd639716a5f6.png)

that's why it will not appear on GitHub

**Reply by mystertaran** [reply]
Okay so we are probably close to solve this problem. There is some option to synchronize my GitHub profile with Git Desktop? I am asking, because I made this repository (szymontaranczewski.com) by using github.com so now I am not able to publish this repo by using desktop version, because its already exist with that name.

**Reply by jfullstackdev** [reply]
did you create it separately on local using GitHub Desktop ? you should have cloned it from GitHub, not create it in local

**Reply by mystertaran** [reply]
Ohh mate, you are right, I made such a mistake. Thanks for your help and have a nice day!
